client/shipping-label/components/label-purchase-modal/packages-step/package-info.js

```diff
@@ -99,12 +99,12 @@ const PackageInfo = ( {
 
 	const renderPackageSelect = () => {
 		if ( isIndividualPackage ) {
-			return ( <div className="wcc-package__desc">
-				<div className="wcc-package-items-header">
-					<FormLegend className="wcc-package-item__name">{ __( 'Individually Shipped Item' ) }</FormLegend>
+			return ( <div>
+				<div className="packages-step__package-items-header">
+					<FormLegend className="packages-step__package-item-name">{ __( 'Individually Shipped Item' ) }</FormLegend>
```

Comment:
Removed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/9bfc8324520cedbc3983ed46f7905a0cf4be379f